### PR TITLE
Bootstrap Chinese localization for website

### PR DESCRIPTION
**Pull request by tengqm**
While I'm learning the basics of Operator SDK, I read the website and asked myself -- how to make this learning process something useful to others. That's the reason behind this effort.

I'll go through all documentation and translate them into Chinese version where appropriate.

**Description of the change:**

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by tengqm** [comment]
Some previews:
image

**Comment by tengqm** [comment]
image

**Comment by tengqm** [comment]
image

**Comment by tengqm** [comment]
image

File: website/config.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 baseURL = "/"
-languageCode = "en-us"
+# languageCode = "en-us"
```
Comment (asmacdo) [inline website/config.toml:2]: This line was copied from an example config, and `<html lang="en" class="no-js">` seems to indicate that the language code is coming from `[languages.en]`. Lets just delete it.
```suggestion
```

File: website/content/zh/build/images/arrow.svg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In  -->
```
Comment (asmacdo) [inline website/content/zh/build/images/arrow.svg:1]: @matthewcarleton can we move these images to a language neutral location like https://github.com/operator-framework/operator-sdk/tree/master/website/static ?

Comment (matthewcarleton) [reply website/content/zh/build/images/arrow.svg:1]: I don't see why not. Do you want me to do it?

File: website/config.toml (hunk 1 above)
Comment (tengqm) [reply website/config.toml:2]: will do.

File: website/content/zh/build/images/arrow.svg (hunk 2 above)
Comment (tengqm) [reply website/content/zh/build/images/arrow.svg:1]: Done. See comment below.

**Comment by tengqm** [comment]
Changes to the shared assets (images) under the `/content/en/build/images` directory.

The following files are moved to `/static` for sharing across localizations.

- `bg-masthead-SDK.svg`
- `logo-sm.svg`
- `logo.svg`
- `message.png`

The following files are moved from `/content/en/build/images` to `/static/attic` because they don't seem to be referenced anywhere. We may later decide to kill them or fix the references if any.

- `Asset 1.png`
- `Asset 2.png`
- `arrow.png`
- `bg-masthead-green.svg`
- `bg-masthead-purple.svg`
- `bg-masthead.svg`
- `header-bg.svg`
- `ico-build.svg`
- `ico-discover.svg`
- `sprite.svg`

**Comment by camilamacedo86** [comment]
Hi @tengqm,
 
Unfortunately, I believe that none of its owners/admins knows Chinese and would be really hard for our keep maintained many languages.it will need to rely in the community contributions. So, IMO we need clarifies that the English docs are mainly maintained just in case it gets outdated. At least, Chinese users will be able to have the thought/light to try to check the English version if something gets wrong.   

So, wdyt about add in some way the info that any language that is not English is just maintained by the community and can be outdated? In hope that I could clarify my concern and I am open a better way the communicate and let them ware of that. 

@asmacdo @matthewcarleton wdyt?

**Comment by joelanford** [comment]
@camilamacedo86 Your comment is especially relevant given how many content and organizational changes we're in the process of making to the docs right now.

I agree that there is a concern that localizations that are not maintained with code merges can become outdated quickly. On the flip side, I'm 100% behind making operator-sdk more accessible to more people.

What's the best way to solve this?

**Comment by asmacdo** [comment]
I am planning to wait until the [organization PR](https://github.com/operator-framework/operator-sdk/pull/3382) before we move on this (I contacted @tengqm  via slack).

+1 Adding `Engilish (Official)` 

I think we can merge this PR without careful review of the Chinese content to get the bootstrapping in place, and then highlight it on the mailing list-- I hope that this will be helpful to many, and they may be willing to contribute to the effort.

Moving forward, we should allow each localization to work relatively independently. I have more to learn here, but I'd like to use the [Kubernetes localization effort](https://kubernetes.io/docs/contribute/localization/) as a baseline.

**Comment by tengqm** [comment]
Maybe it's not the best moment to introduce localization considering that we are adding/removing/replacing commands so frequently. I'm okay to postpone this effort until we have a more stable tool and English docs site. Maybe after 1.0 is out?

**Comment by camilamacedo86** [comment]
Hi @tengqm, 

I am closing this one because it is requiring to rebase and whiteout interaction for a while. 
However, would be very nice to get this contribution. Please, feel free to re-open and/or make a new pr for that.

**Comment by asmacdo** [comment]
+1. I think we are in a better position to do this now, but please ping one of us in slack first so we can make sure.

**Comment by tengqm** [comment]
I never get a green light that this one is okay to proceed before the PR is closed ... sigh.

**Comment by asmacdo** [comment]
@tengqm I'll bring this up with the other maintainers today to see if there are any major docs changes in the works, I'll follow up with you on slack.

**Comment by tengqm** [comment]
@asmacdo Thanks. Let me know when there is a decision from the team.

**Pull request closed without merging**